README.md

```diff
@@ -324,11 +324,9 @@ This is a list of resources for learning C programming in general, or something
 * [Diving into concurrency: trying out mutexes and atomics][202]
 * [Introduction to OpenMP][207] (video)
 * [OpenMP tutorial][264] (for the OpenMP3 standard)
-* [memcpy vs memmove][205]
```

Comment:
This is a tricky one, since the "invalid certificate" error could be temporary in nature (they seem to be using a self-signed one, which throws a warning in all the browsers).

The content still seems to be there, and since it was good enough to be included in the repo, I don't think that invalid HTTPS certificate is enough of a reason to remove it, since it's still a quality content. 

With all that said, I think it would be a nice solution to change the link to the latest [Wayback machine snapshot](https://web.archive.org/web/20170620131430/https://www.tedunangst.com/flak/post/memcpy-vs-memmove) of the page, as a hopefully temporary measure in order to avoid linking to improperly secured websites. That way, we retain the content and browsers won't complain.

I wonder how the rest of the awesome ecosystem deals with such errors.